Source/SelfService/Web/logging/logFilter/filterSelect.tsx

```diff
@@ -12,9 +39,6 @@ export const FilterSelect = <T,>(props: FilterSelectProps<T>) =>
         variant='standard'
         sx={{
             ...props.sx,
-            '&:before, &:hover:not(.Mui-disabled):before, &:after': {
-                border: 'none',
-            },
-            'typography': 'button',
+            ...styles
```

Comment:
Could you swap the order of these, so the `props.sx` is the last? This way `props.sx` property can be overridden by a developer consuming this component :)